platform/jvm/buildSrc/src/main/kotlin/dependency-license-config.gradle.kts

```diff
@@ -0,0 +1,13 @@
+import com.github.jk1.license.filter.SpdxLicenseBundleNormalizer
+import com.github.jk1.license.render.JsonReportRenderer
+
+plugins {
+    id("com.github.jk1.dependency-license-report")
+}
+
+licenseReport {
+    configurations = arrayOf("releaseRuntimeClasspath")
+    allowedLicensesFile = project.rootProject.file("allowed-licenses.json")
+    renderers = arrayOf(JsonReportRenderer())
+    filters = arrayOf(SpdxLicenseBundleNormalizer())
+}
```

Comment:
^ here we centralize the plugin logic using gradle's precompiled convention plugins https://docs.gradle.org/current/userguide/custom_plugins.html